lib/stripe/stripe_client.rb

```diff
@@ -179,14 +165,10 @@ def execute_request(method, path,
 
       # Encoding body parameters is a little more complex because we may have
       # to send a multipart-encoded body. `body_log` is produced separately as
-      # a variant of the encoded form that's output-friendly. e.g. Doesn't show
-      # as multipart. File objects are displayed as such instead of as their
-      # file contents.
-      body, body_log = if body_params
-                         encode_body(body_params, headers)
-                       else
-                         [nil, nil]
-                       end
+      # a log-friendly variant of the encoded form. File objects are displayed
+      # as such instead of as their file contents.
+      body, body_log =
+        body_params ? encode_body(body_params, headers) : [nil, nil]
```

Comment:
Normally wouldn't go for this style of line golf, but this version is nicely symmetrical with query parameters on the line directly above:

``` ruby
      query = query_params ? Util.encode_parameters(query_params) : nil
```